lib/output/utils.js

```diff
@@ -5,9 +5,9 @@ function isObject(value) {
   return typeof value === "object" && value !== null || typeof value === "function";
 }
 
-function hasOwn(obj, prop) {
-  return Object.prototype.hasOwnProperty.call(obj, prop);
-}
+/** @type {function(*, string | number | symbol): boolean} */
+const hasOwn = Function.prototype.bind.call(Function.prototype.call, Object.prototype.hasOwnProperty);
+Object.defineProperties(hasOwn, { name: { value: "hasOwn" }, length: { value: 2 } });
```

Comment:
No need for this.

Reply:
This&nbsp;ensures&nbsp;that the&nbsp;values of&nbsp;`hasOwn.name` and&nbsp;`hasOwn.length`&nbsp;remain what&nbsp;they&nbsp;are&nbsp;now.